tools/handle_ocp_versions.py

```diff
@@ -11,6 +11,10 @@
 from utils import check_output
 
 OCP_VERSIONS_FILE = "default_ocp_versions.json"
+
+# 0.2.0 SNO was based on 4.7 but was placed under the 4.8 version key in versions JSONs.
+# This entry disables the validation for that mismatch. Safe to remove this line and all
+# of its usages in about a month from now, when this image is less relevant.
```

Comment:
I think you can remove this now that we use 0.6.0 which is based on 4.8 image

Reply:
The integration deployment still uses 0.2.0. I'll remove it in a month once it's obviously no longer a problem

Reply:
> Omer can we have the same skip pull secret validation we had for 0.2.0 image?

Replying here because of some reason I can't reply to this comment:

The method we used to skip pull secret validation is exactly this - add `registry.svc.ci.openshift.org` to the list of public registries